hikari/guilds.py

```diff
@@ -424,6 +424,10 @@ def guild_avatar_url(self) -> typing.Optional[files.URL]:
     @property
     def default_avatar_url(self) -> files.URL:
         return self.user.default_avatar_url
+    
+    @property
+    def display_avatar_url(self) -> files.URL:
+        return self.guild_avatar_url or self.avatar_url or self.default_avatar_url
```

Comment:
```suggestion
        return self.guild_avatar_url or super().display_avatar_url
```

Would help avoid duplication here